blackjack.js

```diff
@@ -19,10 +43,43 @@ const player; // TODO
  * @returns {boolean} blackJackScore.isSoft
  */
 const calcPoints = (hand) => {
-  // CREATE FUNCTION HERE
+  // CREATE FUNCTION HERE 
+  total = 0;
+  let calcHand = [hand];
 
+  for(let h=0; h < calcHand.length; h++){
+    switch(true){
+      case ({displayVal} === 'Ace'):
```

Comment:
Ah, okay I see a slight criss cross on syntax that is breaking this, but I think I know the intent. This usage of the `{displayVal}` seems to be wanting to destructure the value and pull it from the current card of the for loop. That's a good approach, but needs to be done in a different place. Situated here in the switch's `case` it just seems broken. 

What you want to do is settle the values before the switch . . . so  first thing in your for loop, grab the values like so : 
`const {val, displayVal} = calcHand[h];`
_then_ do the switch with just `val` and `displayVal`

Now in you case statements you can just get rid of those `{}` that don't have a valid syntax meaning as such. This will fix some of the `undefined` errors that happen.